app/views/admin/courses/_form.html.erb

```diff
@@ -14,5 +14,10 @@
         <%= f.text_field :slug %>
     </div>
 
+    <div class="field">
+        <%= f.label "Sort by number of previous resolved requests" %>
```

Comment:
It might be helpful to include a time period in this label - is this 'sort by number of previously resolved requests over all time'?

Reply:
Updated version:
<img width="493" alt="Screen Shot 2019-05-03 at 12 36 53 AM" src="https://user-images.githubusercontent.com/12666146/57119941-d2e17600-6d3b-11e9-84b3-8381477344de.png">